### ISSUE TITLE
attaching pgfe connection to event ports

### ISSUE THREAD

**Issue by kvxmmu** [dmitigr/pgfe#14]
How can i attach pgfe to existing event loop based on epoll?

**Comment by dmitigr** [comment]
Hello,

`Connection::socket()` shall be exposed for that. Is it still actual?

**Comment by kvxmmu** [comment]
@dmitigr No, at least for me, but if this is implemented it would be great

**Comment by dmitigr** [comment]
> @dmitigr No, at least for me, but if this is implemented it would be great

@kvxmmu Ok, I see. I've planned asynchronous IO API (with automatic polling via libuv and callbacking) in v2.1. Could you please describe the use case for this from your practice? Thanks.

**Comment by kvxmmu** [comment]
@dmitigr Yes, currently I am developing NeoGrok(https://github.com/kvxmmu/neogrok, TL;DR: dedicated ngrok alternative) and in the past it was implemented in C++ and event port was epoll(legacy-server branch). I wanted to implement authorization through database, and not through universal password, so lack of connection interface was a problem.

**Comment by dmitigr** [comment]
@dmitigr Ok, I see. Thank you!
